instrumentation-api-semconv/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/url/UrlParserTest.java

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.instrumenter.url;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+class UrlParserTest {
+
+  @Test
+  public void testGetHost() {
```

Comment:
Nit: JUnit 5 tests can be package private
```suggestion
  void testGetHost() {
```